DEVELOPMENT.md

```diff
@@ -47,6 +47,29 @@ Spec changes should be approved by a majority of the committers.  Approval can b
 
 No change should be approved until there is documentation for it, supplied in an accompanying PR.
 
+## Draft Features
+
+Where suitable, features will be introduced as draft but OAI approved extensions.
+By introducing new features this way we enable new features to be designed, documented and then implemented by tools that are interested in the feature, without putting the burden of implementation on all tooling.
+If the feature is successfully implemented and there is demonstrable value added by the feature, it will become a candidate for inclusion in a future release of the specification, at which point all tools will be expected to support the feature.
+
+Draft feature extensions are identified by the `x-oas-draft-` prefix and can only be used where existing extensions are permitted.
+This ensures no existing tooling will affected by the introduction of the draft feature.
+If the feature is deemed appropriate for inclusion in the OAS, the `x-oas-draft-` prefix will be removed.
+Tooling that supports draft features should plan for the future removal of the prefix.
```

Comment:
So the final property name will usually (or always?) be the same as the draft property name, minus the `x-oas-draft-` prefix, correct? If tool developers can rely on this, at least as the usual/expected case, it makes it somewhat easier to manage the lifecycle.